package/gluon-mesh-vpn-wireguard/files/usr/bin/wg-broker

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+run_broker() {
+  local interface="$1"
+  local pubkey="$2"
+  local port
+  local interval=5
+
+  group_peers=$(uci show wireguard|grep mesh_vpn_backbone_peer|grep =peer|cut -d"." -f2|cut -d= -f1)
+
+  for i in $group_peers
+  do
+    if [ $interface == $(uci get wireguard.$i.ifname) ]; then
+      thispeer=$i
+      break;
+    fi
+  done
+
+  localtime=$(date +%s)
+  if [[ -n $thispeer ]]; then
+
+    remote=$(uci get wireguard.$thispeer.remote)
+    brokerport=$(uci get wireguard.$thispeer.broker_port)
+    localport=$(wg show $interface listen-port)
+
+    serverreply="$(nc_timeout 10 $remote $brokerport "{\"pubkey\":\"$pubkey\", \"localport\":$localport }")"
```

Comment:
Please add an API-Version here.

Reply:
there actually is an api version in the server code already. I am not checking it anywhere just yet. Checks can be added when breaking changes happen.